NeoML/src/Dnn/Rowwise/ChannelwiseWith1x1.cpp

```diff
@@ -44,15 +44,19 @@ CRowwiseChWith1x1::CRowwiseChWith1x1( IMathEngine& mathEngine ) :
 
 CRowwiseOperationDesc* CRowwiseChWith1x1::GetDesc()
 {
-	CConstFloatHandle channelwiseFreeTermData = channelwiseFreeTerm == nullptr ? CConstFloatHandle()
+	auto chFreeTermData = ( channelwiseFreeTerm == nullptr ) ? CConstFloatHandle{}
```

Comment:
what's the  reason to change anything here?

Reply:
reverted